src/Bridge/Symfony/XliffFileDumper.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony;
+
+use Symfony\Component\Translation\Dumper\FileDumper;
+use Symfony\Component\Translation\MessageCatalogue;
+
+/**
+ * Custom XliffFileDumper for use with phrase app
+ */
+class XliffFileDumper extends FileDumper
```

Comment:
I do use tags for the domain (for filtering translations by domain f.e.). The problem is, that for the translation id (`md5($source)`) is used in symfony, which results in a unreadable translation key. You do not know what you want to translate, when you see the phraseapp website. The `source` value of the xliff file is ignored by phraseapp - only translation id is visible and used as key.